docs/mindsdb_sql/sql/create/jobs.mdx

```diff
@@ -21,22 +21,24 @@ Here is the syntax:
 
 ```sql
 CREATE JOB [IF NOT EXISTS] [project_name.]job_name [AS] (
-   <mindsdb_sql_query_1>[; <mindsdb_sql_query_2>]
+   <query_1>[; <query_2>][; ...]
 )
 [START <date>]
 [END <date>]
-[EVERY [number] <period>];
+[EVERY [number] <period>]
+[IF (<query_1>[; <query_2>][; ...])];
 ```
 
 Where:
 
 | Expression                                          | Description                                                                                                                                                         |
 | --------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `[project_name.]job_name`                           | Name of the job preceded by an optional project name where the job is to be created.                                                                                |
-| `<mindsdb_sql_query_1>[; <mindsdb_sql_query_2>]`    | One or more queries separated by `;` to be executed by the job.                                                                                                     |
+| `[project_name.]job_name`                           | Name of the job preceded by an optional project name where the job is to be created. If you do not provide the `project_name` value, then the job is created in the default `mindsdb` project. |
+| `<query_1>[; <query_2>][; ...]`                     | One or more queries separated by `;` to be executed by the job.                                                                                                     |
 | `[START <date>]`                                    | Optional. The date when the job starts its periodical or one-time execution. If not set, it is the current system date.                                             |
 | `[END <date>]`                                      | Optional. The date when the job ends its periodical or one-time execution. If it is not set (and the repetition rules are set), then the job repeats forever.       |
-| `[EVERY [number] <period>]`                         | Optional. The repetition rules for the job. If not set, the job runs once, not considering the end date value. If the `number` value is not set, it defaults to 1. |
+| `[EVERY [number] <period>]`                         | Optional. The repetition rules for the job. If not set, the job runs once, not considering the end date value. If the `number` value is not set, it defaults to 1.  |
+| `[IF (<query_1>[; <query_2>][; ...])]`              | Optional. If the query returns one or more rows, only then the job will execute.                                                                                    |
```

Comment:
one correction: only if the last query in sequence (<query_1>[; <query_2>][; ...]) returns >0 rows job will execute